apps/api/src/domain/common/entities/multilingual-text.ts

```diff
@@ -173,6 +173,9 @@ export class MultilingualText extends BaseDomainModel implements IMultilingualTe
     translate(itemDto: DTO<MultilingualTextItem>): ResultOrError<this> {
         const item = new MultilingualTextItem(itemDto);
 
+        // TODO
```

Comment:
Remove this. It's already there, it's just that it returns errors and leaves it to the client to throw, which I think is the correct approach.